### PR TITLE
Blog post release process

### PR DESCRIPTION
**Pull request by campoy**
Fixes https://github.com/src-d/feature-idea/issues/13

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,21 +11,43 @@ Please,
...
 
 <!-- /TOC -->
 
+## Propose a new blog post
+
+The first step on the process to release a blog post is to reach general agreement on whether the topic
+and general structure of the blog post makes sense. This will help you structure your blog post early and
+avoid wasting any time by receiving feedback too late.
+
+We use a very simila approach to [src-d/conferences](https://github.com/src-d/conferences), using issues
```
Comment (dpordomingo) [inline CONTRIBUTING.md]: s/simila/similar

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,21 +11,43 @@ Please,
...
+and PRs as a way to track and approve proposals.
+
+Whenever you have a new idea for a blog post simply create a new issue following the
+[post issue template](.github/ISSUE_TEMPLATES/post.md), you can also simply click
+[here](https://github.com/src-d/blog/issues/new?template=bugs.md).
```
Comment (vmarkovtsev) [inline CONTRIBUTING.md]: bugs.md?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,21 +11,43 @@ Please,
...
+[![propose new blog post](https://svg-badge.appspot.com/badge/PROPOSE%20NEW%20BLOG%20POST/click?color=fea)](https://github.com/src-d/blog/issues/new?template=bugs.md)
+
+Once you've created this issue make sure to add the `post` label and assign it to `@campoy`,
+he will then review the proposal and give feedback on it until the initial proposal is accepted.
+Once the proposal has been accepted, the issue will be labeled with an `accepted` label, or if it's
+been rejected or abandoned the issue will be closed.
+
+Once accepted, you can start writing your blog post, creating a new pull request that will `fix` the issue
+you created for the proposal. Make sure you add `[WIP]` at the beginning of the title in the PR until you're
+ready for review.
```
Comment (vmarkovtsev) [inline CONTRIBUTING.md]: Below: `make serve;` - should be no semicolon in the end

Comment (vmarkovtsev) [inline CONTRIBUTING.md]: Below: peer review: "vmarkovtsev on content". Should be @campoy instead! So happy I do not have to review posts now, yessssss

**Comment by vmarkovtsev** [comment]
@campoy There is no info about the promotion phase: Twitter, Reddit, HN, who, when, [healthy teeth](https://src-d.slack.com/archives/C024F567F/p1499936793509195), etc.

File: CONTRIBUTING.md (hunk 3 above)
Comment (campoy) [reply CONTRIBUTING.md]: this is not something I wrote, but I'll fix it

Comment (campoy) [reply CONTRIBUTING.md]: fixed, and congrats 🎉

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,21 +11,43 @@ Please,
...
+avoid wasting any time by receiving feedback too late.
+
+We use a very similar approach to [src-d/conferences](https://github.com/src-d/conferences), using issues
+and PRs as a way to track and approve proposals.
+
+Whenever you have a new idea for a blog post simply create a new issue following the
+[post issue template](.github/ISSUE_TEMPLATES/post.md), you can also simply click
+[here](https://github.com/src-d/blog/issues/new?template=post.md).
+
+[![propose new blog post](https://svg-badge.appspot.com/badge/PROPOSE%20NEW%20BLOG%20POST/click?color=fea)](https://github.com/src-d/blog/issues/new?template=post.md)
```
[the remark this answers is not in the document]
Comment (campoy) [reply CONTRIBUTING.md]: Sure, that sounds like a good idea. I'll add this.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -171,3 +202,18 @@ The blog is published automatically following the source{d} [Continous Delivery
 For any blog post to be published, it must follow the conventions given by this guide, and the following technical ones regarding the post [front matter](#front-matter):
 - `draft` key must be unset (or at least it must be `false`),
 - `date` key must be set to before &ndash;or equals to&ndash; the deploy date
+
+## Publishing and promoting a blog post
+
+Once the blog post is ready for publication we'll schedule the publication for a specific time.
+This will probably be on a Tuesday at 8am, but different options can be negotiated too.
```
Comment (vmarkovtsev) [inline CONTRIBUTING.md]: 8am which time zone?

Comment (campoy) [reply CONTRIBUTING.md]: fixed

**Pull request merged**